mri/operators/fourier/non_cartesian.py

```diff
@@ -494,18 +498,25 @@ def __init__(self, kspace_loc, shape, implementation='cpu', n_coils=1):
     def _op(self, data):
         fft_along_z_axis = np.fft.fftshift(np.fft.fft(
             np.fft.ifftshift(data, axes=2),
-            n=self.acq_num_slices,
             norm="ortho"),
             axes=2)
+
         stacked_kspace = np.asarray(
             [self.plane_fourier_operator.op(fft_along_z_axis[:, :, slice])
-             for slice in np.arange(self.acq_num_slices)])
-        stacked_kspace = np.reshape(stacked_kspace,
-                                    self.acq_num_slices * self.stack_len)
+             for slice in range(self.num_slices)])
+        stacked_kspace_sampled = np.zeros((self.acq_num_slices,
```

Comment:
I think this is legacy

Reply:
Yes, we can remove this based on new codes, this seems like a redundant line here